public/index.html

```diff
@@ -44,18 +44,18 @@
 
             <div class="col-sm-12 col-md-12 col-lg-6 mb-3">
                 <div class="card border border-brown shadow">
-                    <div class="card-body text-center">
+                    <a class="card-body text-center" href="https://bscscan.com/token/0x0de08c1abe5fb86dd7fd2ac90400ace305138d5b" style="color: #4f4f4f;">
                         <h4><img src="/images/bsc.svg" style="height:40px;" class="mb-2 me-2">Total Wrapped iDNA issued</h4>
                         <h5 id="total-bsc">- iDNAs</h5>
-                    </div>
+                    </a>
                 </div>
             </div>
             <div class="col-sm-12 col-md-12 col-lg-6 mb-3">
                 <div class="card border border-brown shadow">
-                    <div class="card-body text-center">
+                    <a class="card-body text-center" href="https://scan.idena.io/address/0x98D16d7021930b788135dD834983394fF2De9869" style="color: #4f4f4f;">
```

Comment:
Please use _global_variables.IDENA_EXPLORER_ and _global_variables.IDENA_WALLET_ here

Reply:
Ok will do